lib/moon/components/select/single_select/value/selected_value.ex

```diff
@@ -15,21 +15,18 @@ defmodule Moon.Components.Select.SingleSelect.Value.SelectedValue do
   prop label, :string, default: ""
   prop placeholder, :string, default: ""
   prop size, :string
-  prop class, :css_class, default: ""
 
   def render(assigns) do
     has_value = has_value(assigns.option)
-    has_left_icon = has_value && not is_nil(assigns.option[:left_icon])
-    has_right_icon = has_value && not is_nil(assigns.option[:right_icon])
+    has_left_icon = has_value and not is_nil(assigns.option[:left_icon])
+    has_right_icon = has_value and not is_nil(assigns.option[:right_icon])
     has_icons = has_left_icon or has_right_icon
     is_label = not is_nil(assigns.label)
     is_inner_label = is_label and assigns.size == "xl"
+    has_option_style = has_value and not is_nil(assigns.option[:class])
```

Comment:
Sorry, this does not make sense.

Why we convert string to boolean? Why we even have custom class support here?

Reply:
I'm not convert string to boolean. I check key `class` in option and receive boolean value.
Because `class` is optional